src/Diagnostics.DataProviders/Exceptions/KustoResponseSchemaException.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Diagnostics.DataProviders.Exceptions
+{
+    public class KustoResponseSchemaException : Exception
```

Comment:
The name can create confusions between our ExceptionTypes and Kusto Team ExceptionTypes in future. Instead of calling Kusto, can we use this Exception for any provider in future. How about 'MalformedResponseException' or something like that?